gallery/how_to/deploy_models/deploy_model_on_adreno.py

```diff
@@ -0,0 +1,295 @@
+"""
+Deploy the Pretrained Model on Adreno
+=======================================
+**Author**: Daniil Barinov
+
+This article is a step-by-step tutorial to deploy pretrained Pytorch ResNet-18 model on Adreno (on different precisions).
+
+For us to begin with, PyTorch must be installed.
+TorchVision is also required since we will be using it as our model zoo.
+
+A quick solution is to install it via pip:
+
+.. code-block:: bash
+
+  pip install torch
+  pip install torchvision
+
+Besides that, you should have TVM builded for Android.
+See the following instructions on how to build it.
+
+`Deploy to Adreno GPU <https://tvm.apache.org/docs/how_to/deploy/adreno.html>`_
+
+After the build section there should be two files in *build* directory «libtvm_runtime.so» and «tvm_rpc». 
+Let's push them to the device and run TVM RPC Server.
+"""
+
+
+#################################################################
```

Comment:
Why before you use `"""` for commenting and now `#`? Probably use one comment style for huge multiline comments?